corehq/apps/builds/templates/builds/bootstrap5/edit_menu.html

```diff
@@ -15,103 +15,91 @@
 <div class="alert alert-info"><strong>{% trans "Editing" %} {{ doc.ID }}</strong></div>
 
 
-<div class="card card-default">
-  <div class="card-header">
-    <h3 class="card-title">{% trans "Menu Options" %}</h3>
-  </div>
+<div class="card">
   <div class="card-body">
```

Comment:
I think this would match the original look better.

```
<div class="card-header">{% trans "Menu Options" %}</div>
<div class="card-body">
....
```

Reply:
https://getbootstrap.com/docs/5.3/components/card/#header-and-footer

Reply:
Thanks, that's much nicer. Fixed in 05f3a7cee76